website/content/en/preview/reference/metrics.md

```diff
@@ -38,6 +38,30 @@ Number of nodeclaims disrupted in total by Karpenter. Labeled by reason the node
 Number of nodeclaims created in total by Karpenter. Labeled by reason the nodeclaim was created and the owning nodepool.
 - Stability Level: STABLE
 
+### `operator_nodeclaim_status_condition_transitions_total`
+The count of transitions of a nodeclaim, type and status.
```

Comment:
We should also include the labels which can be found here: https://github.com/awslabs/operatorpkg/blob/main/status/metrics.go